### PR TITLE
Move to fixtures

### PR DESCRIPTION
**Pull request by jettify**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Some simple testing tasks (sorry, UNIX only).
 
 flake:
-	pep8 aiohttp_mako tests
-	pyflakes aiohttp_mako tests
+	pep8 aiohttp_mako tests/test_simple_renderer.py
```
Comment (asvetlov) [inline Makefile]: Is it intentional?

**Comment by jettify** [comment]
Yes, in order to pass pyflakes checks for python 3.5 syntax.

**Comment by jettify** [comment]
oh, I see what you are mean, I will fix

File: Makefile (hunk 1 above)
Comment (jettify) [inline Makefile]: fixed in master

**Pull request merged**